glow/src/text.rs

```diff
@@ -12,15 +12,21 @@ pub struct Pipeline {
 
 impl Pipeline {
     pub fn new(gl: &glow::Context, default_font: Option<&[u8]>) -> Self {
-        // TODO: Font customization
-        let font_source = font::Source::new();
+        let default_font = default_font.map(|slice| slice.to_vec());
 
-        let default_font =
-            default_font.map(|slice| slice.to_vec()).unwrap_or_else(|| {
+        // TODO: Font customization
+        #[cfg(feature = "default_system_font")]
+        let default_font = {
+            let font_source = font::Source::new();
```

Comment:
We may be able to move this line inside the `or_else` closure. This way we avoid creating a `Source` when the user has already provided a `default_font`.